### ISSUE TITLE
Rasterizer breaks when every contour contains only one unique point

### ISSUE THREAD

**Issue by astiob** [libass/libass#303]
@MrSmile

```
{\p1}m 0 0 l 0 0
```

with any additional number of `l 0 0` appended, and with any additional number of similar single-unique-point contours appended

produces this message:

```
[ass] Glyph bounding box too large: -67108863x-67108863px
```

and the drawing’s rasterization is aborted. The drawing itself is invisible anyway, but if it has a nonzero border and `BorderStyle` is 3, a border box should be visible, but it isn’t rasterized because of this error.

Altering any contour to make it have more than one unique point magically fixes the entire drawing.

**Comment by MrSmile** [comment]
Rasterizer skips any zero-length lines so in this case it reports that "nothing to draw" but that's not handled in `outline_to_bitmap()` correctly. I've fixed that in https://github.com/libass/libass/pull/285 but only for one function. Subsequent code treats empty base bitmap as error and skips the following stages. I'll try to fix PR to remove that dependency.

**Comment by astiob** [comment]
Do I understand correctly that in #285 you remove the warning but it still returns a null bitmap, and you’ll now add a fix that makes it continue despite the null base bitmap?

**Comment by MrSmile** [comment]
Yes, exactly. It's simple to do in PR as I already removed most of existing checks for null bitmaps.

**Comment by astiob** [comment]
OK, cool.

<div>-------- Исходное сообщение --------</div><div>От: MrSmile <notifications@github.com> </div><div>Дата:12.03.2018  3:45  (GMT+02:00) </div><div>Кому: libass/libass <libass@noreply.github.com> </div><div>Копия: Oleg Oshmyan <chortos@inbox.lv>, Author <author@noreply.github.com> </div><div>Тема: Re: [libass/libass] Rasterizer breaks when every contour contains
  only one unique point (#303) </div><div>
</div>Yes, exactly. It's simple to do in PR as I already removed most of existing checks for null bitmaps.

**Comment by TheOneric** [comment]
It seems this was fixed with the merge of #285.
I can reproduce the error message with release 0.14.0, but not with latest git master e514062 .
